frontend/src/views/dashboard/index.js

```diff
@@ -51,6 +56,100 @@ const ConnectedAssignmentsList = connect(state => ({
     assignments: assignmentsSelector(state)
 }))(AssignmentsList);
 
+const ConnectedSearchBox = connect(state => ({
+    data: applicantsSelector(state)
+}))(SearchBox);
+
+const ConnectedEmailButton = connect(state => ({
+    data: state.ui.offerTable.selectedIds
+}))(EmailButton);
+
+const ConnectedOfferTable = connect(
+    null,
+    { sendSelectedRows }
+)(CustomTable);
+
+const COLUMNS = [
+    { Header: "First Name", accessor: "first_name", width: 100 },
+    { Header: "Last Name", accessor: "last_name", width: 100 },
+    { Header: "Email", accessor: "email", width: 250 },
+    { Header: "Position title", accessor: "position_title", width: 130 },
+    {
+        Header: "First Time?",
+        accessor: "first_time_ta",
+        Cell: props => <span>{props.value.toString().toUpperCase()}</span>,
+        width: 100
+        // filterMethod: (filter, row) =>
+        //             row[filter.id].startsWith(filter.value) &&
+        //             row[filter.id].endsWith(filter.value)
+    }, // boolean
+    { Header: "Status", accessor: "status", width: 100 },
+    { Header: "Nag Count", accessor: "nag_count", width: 100 }
+];
+
+const DATA = [
```

Comment:
This is temporary data, right? Are you able to use real data now? If you're going to commit temporary data, put a 

```javascript
// XXX this is temporary data and should be removed when issue #... is resolved
```
comment